### PR TITLE
Lazy load resource parameters

### PR DESCRIPTION
**Pull request by jfthuillier**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | api-doc-parser#42
| License       | MIT
| Doc PR        | 

This PR must be merged first : https://github.com/api-platform/api-doc-parser/pull/46

### CODE REVIEW

File: src/hydra/HydraAdmin.js [hunk 1]
```diff
@@ -33,12 +33,26 @@ export default class extends Component {
...
       .apiDocumentationParser(this.props.entrypoint)
       .then(
-        ({api, customRoutes = []}) => ({
-          api,
-          customRoutes,
-          hasError: false,
-          loaded: true,
-        }),
+        ({api, customRoutes = []}) => {
+          const promises = [];
```
Comment (mauchede) [inline src/hydra/HydraAdmin.js]:
```js
const promises = [];
api.resources.forEach(resource => {
    promises.push(resource.getParameters());
});
```

->

```js
const promises = api.resources.map(resource => resource.getParameters());
```

Comment (jfthuillier) [reply src/hydra/HydraAdmin.js]: Done

**Comment by comxd** [comment]
api-platform/api-doc-parser#46 has been merged, can you merge this PR?

**Comment by alanpoulain** [comment]
Shouldn't we wait a release of `api-doc-parser` before though?

**Comment by jfthuillier** [comment]
It has been merged & tagged on 0.7.0 by @mauchede.
Do i need to update `api-doc-parser` on this PR, do i open a new one or is somebody responsible of upgrading depencies ?

**Comment by alanpoulain** [comment]
Since it's a dependency, I think it should be done on this PR.

**Comment by jordanaubert** [comment]
Related issue https://github.com/api-platform/admin/issues/169

**Pull request merged**